CMakeLists.txt

```diff
@@ -2233,7 +2235,7 @@ add_library(MP3GuessEnc STATIC EXCLUDE_FROM_ALL
   lib/mp3guessenc-0.27.4/bit_utils.c
 )
 if(WIN32)
-  target_compile_definitions(MP3GuessEnc PRIVATE __WINDOWS__)
+  target_compile_definitions(MP3GuessEnc PRIVATE __WINDOWS__ _CRT_SECURE_NO_WARNINGS)
```

Comment:
The same here. Is it possible to disable the warning conditional near the effected code?

Reply:
We use this already for our own code. It inhibits the MSVC security warnings when you use string functions without the _s suffix. A C++ standard extension which only Microsoft implemented, because all the other compiler developer  do not agree, that these functions improve the code security.
With _CRT_SECURE_NO_WARNINGS defined, MSVC behaves like gcc and clang.

Reply:
OK, thank you.